tests/common.go

```diff
@@ -3263,7 +3263,7 @@ func DeleteBackupLocation(name string, backupLocationUID string, orgID string, D
 }
 
 // DeleteSchedule deletes backup schedule
-func DeleteSchedule(backupScheduleName, backupScheduleUID, OrgID string) error {
+func DeleteSchedule(backupScheduleName, backupScheduleUID, clusterName, OrgID string) error {
```

Comment:
This might be out of scope for this PR. How about we pass ctx as well?

Reply:
Thanks.

Reply:
It good have the ctx as well, Included it.
Thanks for suggestion.